### PR TITLE
feat: new OnchainAvatar component

### PR DESCRIPTION
**Pull request by alvaroraminelli**
**What changed? Why?**

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

**Comment by alvaroraminelli** [comment]
- missing comments
- missing tests
- missing docs
- review edge cases in the OnchainAvatar

File: README.md [hunk 1]
```diff
@@ -433,6 +433,29 @@ type UseName = {
...
+```tsx
+import { Avatar } from '@coinbase/onchainkit';
+
+<Avatar address="0x1234567890abcdef1234567890abcdef12345678" />;
+```
+
+**@Props**
+
+```ts
+type UseAvatar = {
```
Comment (Zizzamia) [inline README.md:449]: nit for follow up PR, I believe `UseAvatar` should just be `Avatar`.

Also we should start get in the habit to showcase the return html content. under props

**Review by Zizzamia — APPROVED** [review]
Love this first approach!

Great work
